examples/react-native/storybook/addons.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
-
 import '@storybook/addon-actions/register';
 import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
```

Comment:
- `@storybook/addon-knobs` is not in the `package.json` as of today f7d5e65aee9b46787bcc7accd29a197dca55dc6f
- And `@storybook/addon-knobs` version `^6.1` (the one used for the addons) doesn't have a module called

Reply:
FYI @oblador